pkg/controllers/vdb/offlineupgrade_reconciler.go

```diff
@@ -58,6 +58,7 @@ var OfflineUpgradeStatusMsgs = []string{
 	"Shutting down cluster",
 	"Rescheduling pods with new image",
 	"Restarting cluster with new image",
+	"Reinstalling default packages after the first restart of all subcluster(s)",
```

Comment:
Suggest something a bit more terse:
```suggestion
	"Reinstalling default packages",
```